mLRS/Common/common.h

```diff
@@ -24,6 +24,7 @@
 #include "buzzer.h"
 #include "fan.h"
 #include "leds.h"
+//#include "time_stats.h" // Uncomment for time measurement functions
```

Comment:
couldn't we just always have it included?
does it need to be included here, or could it also be included in main?

Reply:
This just seemed like a convenient place to include it.  I left it commented out because it might be used only by developers and I didn't want it taking up space if it's not used when USE_DEBUG is defined.  I'd suggest you decide where to include it.

Reply:
If we always include it, I can add an extra ifdef so it can be disabled independently from USE_DEBUG

Reply:
ok.

Reply:
I guess I would suggest to completely remove the #else with content. That way code needs to be cleaned if not in debug.

Reply:
OK, but I'll remove the empty functions with "#if 0" in case we decide we want to be able to leave some use of these in production.

Reply:
OK.  It's easy enough to put back later if we decide it's handy to measure something whenever debug is on.